.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--
+For Work In Progress Pull Requests, please use the Draft PR feature,
+see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+
+Before submitting a Pull Request, please ensure you've read the CRiSp
+[Contributing Guide](https://cityriverspaces.github.io/CRiSp/CONTRIBUTING.html)
+and [Code of Conduct](https://cityriverspaces.github.io/CRiSp/CODE_OF_CONDUCT.html)
+-->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description
+
+## Related Issues
+
+<!--
+See [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+-->
+
+- Related Issue #
+- Closes #
+
+## Added/updated tests?
+_We encourage you to keep the code coverage percentage at 75% and above._
+
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
+      have not been included_
+- [ ] I need help with writing tests
+
+## Added entry in changelog?
+_For user-facing changes, add a line describing the changes in NEWS.md_
+
+- [ ] Yes
+![alt_text](gif_link)
```

Comment:
Is this something that is here by mistake?
```suggestion
```

Reply:
Yes, that's a mistake. Thanks!